pkg/virtctl/scp/scp.go

```diff
@@ -116,3 +131,42 @@ func usage() string {
   # Copy a file from the remote location to a local folder
   {{ProgramName}} scp jdoe@testvmi:myfile.bin ~/myfile.bin`
 }
+
+func ParseArguments(source, destination string) (LocalArgument, RemoteArgument, bool, error) {
+	var local LocalArgument
+	var remote RemoteArgument
+	var toRemote bool
+	var err error
+
+	toRemote = false
```

Comment:
```suggestion
	toRemote := false
```

Alternatively, drop L138.